test/sample-test/run_sample_test.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2019 Google LLC
```

Comment:
Is this going to be the entrypoint, replacing the run_test.sh?
If so, please also update the dockerfile test/sample-test/Dockerfile.

Reply:
Never mind. I thought the run_test.sh is deleted.